src/vivarium_census_prl_synth_pop/constants/paths.py

```diff
@@ -7,18 +7,19 @@
 BASE_DIR = Path(vivarium_census_prl_synth_pop.__file__).resolve().parent
 
 PROJECT_ROOT = Path(f"/mnt/team/simulation_science/pub/models/{metadata.PROJECT_NAME}")
-ARTIFACT_ROOT = PROJECT_ROOT / "artifacts/"
+ARTIFACT_ROOT = PROJECT_ROOT / "artifacts"
 MODEL_SPEC_DIR = BASE_DIR / "model_specifications"
 RAW_DATA_ROOT = BASE_DIR / "data" / "raw_data"
 
-RAW_RESULTS_DIR_NAME = Path("raw_results")
-FINAL_RESULTS_DIR_NAME = Path("final_results")
+RAW_RESULTS_DIR_NAME = "results"
+FINAL_RESULTS_DIR_NAME = "final_results"
 
-BEST_DIR_NAME = Path("best")
-LATEST_DIR_NAME = Path("latest")
+PROCESSED_RESULTS_DIR_NAME_BASE = "pseudopeople_input_data_usa"
```

Comment:
This is the only new object I added to paths. The rest of the changes are just my ocd doing its thing